index.html

```diff
@@ -122,8 +122,10 @@ <h3>
       RTP header extensions.
     </p>
     <pre class="idl">
-partial dictionary RTCRtpHeaderExtensionCapability {
-  RTCRtpTransceiverDirection direction = "sendrecv";
+// This definition replaces that of webrtc-pc, making `uri` required.
+dictionary RTCRtpHeaderExtensionCapability {
+  required DOMString uri;
```

Comment:
Technically we should make a webrtc-pc PR for making uri required and then have this continue to be a partial dictionary only adding direction. I'm happy to provide a webrtc-pc PR for that and update this PR.

As far as I can tell, making it required in webrtc-pc is compatible because the RTCRtpHeaderExtensionCapability is only ever used as a return value in webrtc-pc

Reply:
But let's get agreement on this first

Reply:
WebRTC PR here: https://github.com/w3c/webrtc-pc/pull/2841
Changed this back to partial dictionary